src/math/Box3.js

```diff
@@ -159,6 +159,12 @@ Box3.prototype = {
 
 					}
 
+				} else if ( node.isSprite ) {
```

Comment:
Only a few lights have proper world space locations, so you would have to limit it to point/spot and rectangle lights otherwise things would get a bit random.

I was aiming to restore the historical behaviour to some extent (although the Object position obviously doesn't  handle the size of a sprite, so it isn't quite accurate), also I think Group() objects might cause some unexpected results.